src/components/cats/cat.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
This file can probably just be `index.js`, so you can then just `import` the folder and it will default to `index.js`